### PR TITLE
Explore page

### PR DESCRIPTION
**Pull request by Megha-Pathak**
Add Explore Page 
- Including Responsive sidebar 
- Card List 
- More Modal 

Preview URL - https://queer-tv-git-explore-page-megha-pathak.vercel.app/explore

### CODE REVIEW

File: src/frontend/components/ExploreVideoCard/ExploreVideoCard.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import Moment from "react-moment";
+import { MoreModal } from "../MoreModal/MoreModal";
```
Comment (suyashpradhan) [inline src/frontend/components/ExploreVideoCard/ExploreVideoCard.js:4]: Can we give a different name for this component? MoreModal looks too generic

File: src/frontend/components/ExploreVideoCard/ExploreVideoCard.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            setIsDropdownMenuOpen={setIsDropdownMenuOpen}
+          />
+        </div>
+      </div>
+      <div className="vertical-video-card-secondary">
+        <span className="vertical-video-card-views">
+          {nFormatter(views, 1)} views
+        </span>
+        <span className="material-icons vertical-video-card-secondary-separator">
+          fiber_manual_record
```
Comment (suyashpradhan) [inline src/frontend/components/ExploreVideoCard/ExploreVideoCard.js:40]: What's this?

File: src/frontend/components/MainSidebar/MainSidebar.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
+import React, { useRef } from "react";
+import { NavLink } from "react-router-dom";
+import { useSidebar } from "../../contexts";
+import { useOnClickOutside, useWindowSize } from "../../hooks";
+import "./MainSidebar.css";
+
+export const MainSidebar = () => {
+  const MainSidebarRef = useRef();
```
Comment (suyashpradhan) [inline src/frontend/components/MainSidebar/MainSidebar.js]: Make use camelCase here

File: src/frontend/components/MainSidebar/MainSidebar.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      setShowSidebar(false);
+    }
+  };
+
+  return (
+    <div ref={MainSidebarRef} className="expanded-sidebar">
+      <header className="sidebar-header">
+        <section>
+          <button className="nav-menu" onClick={() => setShowSidebar(false)}>
+            <span className="material-icons nav-menu-icon">close</span>
```
Comment (suyashpradhan) [inline src/frontend/components/MainSidebar/MainSidebar.js:33]: If the className is given as `material-icons` then why it is a text?

File: src/frontend/components/ToggledSidebar/ToggledSidebar.css [hunk 5]
```diff
@@ -0,0 +1,27 @@
+.shrinked-sidebar {
```
Comment (suyashpradhan) [inline src/frontend/components/ToggledSidebar/ToggledSidebar.css:1]: Component name is ToggledSidebar then change the css related to it as well.

File: src/frontend/components/ToggledSidebar/ToggledSidebar.css [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+.shrinked-sidebar-link {
+  cursor: pointer;
+  padding: 0.875rem;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  aspect-ratio: 1 / 1;
```
Comment (suyashpradhan) [inline src/frontend/components/ToggledSidebar/ToggledSidebar.css:16]: Why aspect ratio is mentioned everywhere?

**Review by suyashpradhan — CHANGES_REQUESTED** [review]
Looks good, added a few commets
Suggestion: Make use .jsx file names if you are working with JS and .tsx for Typescript based files.

File: src/frontend/components/ToggledSidebar/ToggledSidebar.css (hunk 6 above)
Comment (Megha-Pathak) [reply src/frontend/components/ToggledSidebar/ToggledSidebar.css:16]: it is based on this - https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio

File: src/frontend/components/MainSidebar/MainSidebar.js (hunk 4 above)
Comment (Megha-Pathak) [reply src/frontend/components/MainSidebar/MainSidebar.js:33]: We give material icons class and then give text of name of icon we need - this is based on Google action icons

File: src/frontend/components/MainSidebar/MainSidebar.js (hunk 3 above)
Comment (Megha-Pathak) [reply src/frontend/components/MainSidebar/MainSidebar.js]: Done

**Pull request merged**